concepts/05 UI Components/Scheduler/040 Resources/040 Customize Resource Headers.md

```diff
@@ -97,175 +97,7 @@ For AngularJS and Knockout apps, DevExtreme provides the [dxTemplate](/api-refer
-        });
-
-[note] The `dx-item-alias` directive specifies the variable that is used to access the item object.
-
-#####**Knockout**
-
-    <!--HTML--><div data-bind="dxScheduler: {
-        dataSource: schedulerData,
-        currentDate: currentDate,
-        resources: [ roomResource ],
-        groups: [ 'roomId' ],
-        resourceCellTemplate: 'header'
-    }">
-        <div data-options="dxTemplate: { name: 'header' }">
-            <i style="color: blue" data-bind="text: text"></i>
-        </div>
-    </div>
-
-    <!--JavaScript-->var viewModel= {
-        schedulerData: [{
-            text: "Meeting",
-            startDate: new Date("2016-04-24T09:10:00.000Z"),
-            endDate: new Date("2016-04-24T11:20:00.000Z"),
-            roomId: 1
-        },
-        // ...
-        ],
-        roomResource: {
-            fieldExpr: 'roomId',
-            dataSource: [
-                { id: 1, text: 'Room101', color: 'green' },
-                { id: 2, text: 'Room102', color: 'red' },
-                // ...
-            ]
-        },
-        currentDate: new Date(2016, 4, 24)
-    };
-
-    ko.applyBindings(viewModel);
-
-##### Vue
-
-    <!-- tab: App.vue -->
-    <template>
-        <DxScheduler
-            :data-source="appointments"
-            :current-date="currentDate"
-            :groups="groups"
-            resource-cell-template="resourceCellTemplate"
-        >
-            <DxResource
-              :data-source="resources"
-              field-expr="roomId"
-            />
-            
-            <template #resourceCellTemplate="{ data }">
-                <i style="color: blue">{{data.text}}</i>
-            </template>
-        </DxScheduler>
-    </template>
-    <script>
-    import 'devextreme/dist/css/dx.light.css';
-
-    import { DxScheduler, DxResource } from 'devextreme-vue/scheduler';
-
-    export default {
-        components: {
-            DxScheduler,
-            DxResource
-        },
-        data() {
-            return {
-                currentDate: new Date(2016, 4, 24),
-                appointments: [{
-                    text: "Meeting",
-                    startDate: new Date("2016-04-24T09:10:00.000Z"),
-                    endDate: new Date("2016-04-24T11:20:00.000Z"),
-                    roomId: 1
-                }, 
-                // ...
-                ],
-                resources: [
-                    { id: 1, text: "Room101", color: "green" },
-                    { id: 2, text: "Room102", color: "red" },
-                    // ...
-                ],
-                groups: ["roomId"]
-            }
-        }
-    }
-    </script>
-
-##### React
+    Templates allow you to customize UI components. Implement a callback function with custom template and assign it to the [resourceCellRender](/Documentation/ApiReference/UI_Components/dxScheduler/Configuration/#resourceCellRender) property as shown in the following code.
```

Comment:
```suggestion
    Templates allow you to customize UI components. Implement a callback function with custom template and assign it to the [resourceCellRender](/Documentation/ApiReference/UI_Components/dxScheduler/Configuration/#resourceCellRender) property as shown in the following code.
```
"Implement a callback function with custom template " this part doesn't make sense; is there something missing here? can you rewrite it? the relationship of the callback function and the custom template is not clear.

Reply:
DevExtreme templates allow you to customize UI components. Implement a custom template and assign it to the **resourceCellRender** property as shown in the following code.